### PR TITLE
wordvecspace interact feature

### PR DESCRIPTION
**Pull request by RamanjaneyuluIdavalapati**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def get_long_description():
     long_description = ""
 
     for d in dirs:
-        rst_readme = os.path.join(d, "README.rst")
+        rst_readme = os.path.join(d, "README.md")
```
Comment (prashanthellina) [inline setup.py]: @RamanjaneyuluIdavalapati @prasannababuAddagiri  Why this change? It appears the code is looking for an "rst" file.

File: setup.py [hunk 2]
```diff
@@ -42,6 +42,7 @@ def get_long_description():
         'numpy==1.13.1',
         'pandas==0.20.3',
         'numba==0.36.2',
+        'llvmlite==0.21.0',
```
Comment (prashanthellina) [inline setup.py]: @RamanjaneyuluIdavalapati @prasannababuAddagiri Why do we need this new dependency? Is this for "interact" functionality?

File: setup.py (hunk 1 above)
Comment (RamanjaneyuluIdavalapati) [reply setup.py]: As discussed, we will remove long_description.

File: setup.py (hunk 2 above)
Comment (RamanjaneyuluIdavalapati) [reply setup.py]: I am sorry Prashanth. We have mistaken. llvmlite is used by numba. When we install numba==0.36.2, then llvmlite==0.21.0 will automatically install. But on my machine, it didn't happen(the previous llvmlite==0.19.0 is used). I thought that it is problem with numba and we should manually install llvmlite==0.21.0 which is false. I will remove llvmlite.

Comment (prashanthellina) [reply setup.py]: @RamanjaneyuluIdavalapati  No issues. Thank you.
